packages/convex-helpers/README.md

```diff
@@ -630,3 +630,74 @@ export const lastCountLongerThanName = query({
   },
 });
 ```
+
+## Manual Pagination
+
+Note Convex provides built-in pagination through `.paginate()` and
+`usePaginatedQuery()`.
+
+The `getPage` helper gives you more control of the pagination. You can specify
+the index ranges or do multiple paginations in the same query.
```

Comment:
```suggestion
the index ranges or do multiple paginations in the same query.
An index range is all of the documents between two index keys.
An index key is an array of values for the fields in the specified index.
For example, for an index defined like `defineTable({ a: v.number(), b: v.string() }).index("my_index", ["a", "b"])`
an index key might be `[ 3 ]` or `[ 3, "abc" ]`. By default the index is the built-in "by_creation_time" index.
```

Reply:
Sg. We should also mention that index keys returned by getPage are always unique, because they end with the `_id` field